CppCoreGuidelines.md

```diff
@@ -18842,7 +18842,7 @@ Source file rule summary:
 * [SF.4: Include `.h` files before other declarations in a file](#Rs-include-order)
 * [SF.5: A `.cpp` file must include the `.h` file(s) that defines its interface](#Rs-consistency)
 * [SF.6: Use `using namespace` directives for transition, for foundation libraries (such as `std`), or within a local scope (only)](#Rs-using)
-* [SF.7: Don't write `using namespace` at global scope in a header file](#Rs-using-directive)
+* [SF.7: Don't create namespace aliases in header files](#Rs-namespace-aliases)
 * [SF.8: Use `#include` guards for all `.h` files](#Rs-guards)
```

Comment:
A namespace alias is something declared as `namespace MyOrg = LongCompanyName::AwesomeDivision::EpicDivision;", which this rule is not about.  The existing rule (and the modification) is about using directives.